src/team3.html

```diff
@@ -161,9 +161,6 @@
 				type: 'POST',
 				data: { 'word': selectedText, 'language':langaugetowrite },
 			})
-				.done(function (data) {
-					Output.textContent = data;
-				});
 		}
 		else
```

Comment:
Yes, Try and test for the local machine.

If possible can we give a red underline for incorrect words?